### PR TITLE
[SRVKE-1324] Remove matchExpression from webhook definition

### PR DESCRIPTION
**Pull request by pierDipi**
See https://github.com/openshift-knative/serverless-operator/pull/1755 and
https://issues.redhat.com/browse/SRVKE-1324 for more details.

Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

File: openshift/release/generate-release.sh [hunk 1]
```diff
@@ -8,6 +8,7 @@ GITHUB_ACTIONS=true $(dirname $0)/../../hack/update-codegen.sh
 git apply openshift/patches/disable-ko-publish-rekt.patch
 git apply openshift/patches/override-min-version.patch
 git apply openshift/patches/use_kafkacat_from_ocp.patch
+git apply openshift/patches/preserve_match_labels_webhook.patch
```
Comment (aliok) [inline openshift/release/generate-release.sh:11]: Where's this file?

Comment (pierDipi) [reply openshift/release/generate-release.sh:11]: https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.4/openshift/patches/preserve_match_labels_webhook.patch

**Pull request merged**